app/src/main/res/values/strings.xml

```diff
@@ -265,7 +265,9 @@
   <string name="pin_password_hello">Hi, %s!</string>
   <string name="pin_password_admin_enter">Please enter your Administrator PIN.</string>
   <string name="pin_password_user_enter">Please enter your PIN.</string>
-  <string name="pin_password_forgot_pin">I forgot my pin.</string>
+  <string name="pin_password_admin_pin_enter">Administrator’s 5-Digit PIN.</string>
+  <string name="pin_password_user_pin_enter">User’s 3-Digit PIN.</string>
+  <string name="pin_password_forgot_pin">I forgot my pin</string>
```

Comment:
These need more context I think on where the text shows up. For example, the first text could be:

``input_pin_password_as_admin``

Please update all new string names accordingly to have more context.

Reply:
Updated strings.